### PR TITLE
Add documentation for filing and institution status

### LINKED ISSUE #602: Documentation for states
Sorry if these already exist but I couldn't find them.

Like with the [submission states](https://github.com/cfpb/hmda-platform/blob/master/Documents/submission-status.md), it would be good to document the states for both the institutions (see #601) and for the filings (see #600).

### PR DESCRIPTION
**Pull request by kgudel**
Closes #602 

The filing status information is based on [this comment](https://github.com/cfpb/hmda-platform/pull/411#issuecomment-226338262) and [this comment](https://github.com/cfpb/hmda-platform/issues/600#issuecomment-251697290)

The institution status information is based on a preliminary conversation with @lgdean

### CODE REVIEW

File: Documents/institution-status.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+* `status`: `String`
+
+For example:
+
+```json
+      "institution": {
+      "id": "12345",
+      "name": "First Bank",
+      "status": "active"
```
Comment (awolfe76) [inline Documents/institution-status.md]: I think we would want this to be in the same format:
```json
"status": {
    "code": 1,
    "message": "active"
}
```

Comment (kgudel) [inline Documents/institution-status.md]: I was taking the json directly from the API format, which I think that this documentation should follow. If we want to change how the bank status is returned I think that would be the purview of another ticket and the docs could be changed in that PR to match the new API response.

**Comment by kgudel** [comment]
@awolfe76 By adding things about Panel, do you mean adding something like a description of what panel is?

**Comment by awolfe76** [comment]
#601 is doing just that. Its fine if we leave it to that PR to make the change.

For panel, am just wondering if we should add a property to this to show if an institution is in panel or not? Again, another issue is probably best.

👍  from me.

**Comment by awolfe76** [comment]
And sorry for the confusion. Just realized that `inactive` and `active` was in fact talking about panel.

File: Documents/institution-status.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Institution status
+
+Every filing will store a `InstitutionStatus` object with the following structure:
+
+* `status`: `String`
+
+For example:
+
+```json
+      "institution": {
```
Comment (schbetsy) [inline Documents/institution-status.md]: Looks like indentation is a little off here

**Comment by schbetsy** [comment]
Just a quick formatting comment, then this looks good.

Good write-up, @kgudel! Very clear.

File: Documents/filing-status.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "status": {
+    "code": 1,
+    "message": "not-started"
+  }
+}
+```
+
+In order to track the status of a filing for a financial institution, the following states are captured by the backend:
+
+* `1`: `NotStarted` - A filing has been opened but no submission has been submitted for the filing.
```
Comment (awolfe76) [inline Documents/filing-status.md]: Should this be `not-started` to match the message above? Probably the same for the other statuses too.

Comment (kgudel) [inline Documents/filing-status.md]: Can do!

File: Documents/institution-status.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+## Institution status
+
+Every filing will store a `InstitutionStatus` object with the following structure:
```
Comment (schbetsy) [inline Documents/institution-status.md]: Should this say "Every institution", rather than "Every filing"?

Comment (kgudel) [inline Documents/institution-status.md]: Indeed it should!

**Comment by awolfe76** [comment]
👍

**Pull request merged**